utils/rng-tools/files/rngd.init

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh /etc/rc.common
 # Copyright (C) 2011-2014 OpenWrt.org
 
-START=25
+START=0
```

Comment:
Zero seems a bit early.  I don't like the idea of not being able to put something before this...  What about `5`?